crates/bevy_gltf/src/loader.rs

```diff
@@ -593,9 +597,15 @@ async fn load_gltf<'a, 'b, 'c>(
         let mut node_index_to_entity_map = HashMap::new();
         let mut entity_to_skin_index_map = EntityHashMap::default();
         let mut scene_load_context = load_context.begin_labeled_asset();
+
         world
             .spawn(SpatialBundle::INHERITED_IDENTITY)
             .with_children(|parent| {
+                if let Some(extensions) = gltf.extensions() {
+                    parent.spawn(GltfExtensions {
+                        value: serde_json::Value::from(extensions.clone()).to_string(),
```

Comment:
Any reason why we are putting this into the component as a string and not as the already-parsed serde_json::Value? At least with the Value, you have a structure to work with, instead of needing to deserialize it again.

Reply:
Value doesn't implement reflect

Reply:
Github really needs a 🥲 reaction. Can we reflect ignore it? I think that's more useful on the balance: not having to deserialize it seems better than more functional reflection.

Reply:
Hmm we could, but I prefer to have it be a string over reflect ignoring the field, if you feel strongly about this I will change it.

Reply:
Actually, for some reason doing reflect ignore causes it to not get stored when you spawn the scene, so we just can't have it